plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -185,11 +185,23 @@ protected FileCollection parseTarget(Object target) {
 		}
 	}
 
+	private static void relativizeIfSubdir(List<String> relativePaths, File root, File dest) {
+		String relativized = relativize(root, dest);
+		if (relativized != null) {
+			relativePaths.add(relativized);
+		}
+	}
+
+	/**
+	 * Returns the relative path between root and dest,
+	 * or null if dest is not a child of root.
+	 */
+	@Nullable
```

Comment:
I'd argue that the `@Nullable` should be placed between the `static` and `String` on the next line for Java 8 style points, but it's a very small nitpick. Everything else LGTM!